src/components/TableCard.tsx

```diff
@@ -0,0 +1,102 @@
+import { ReactNode, useMemo } from "react";
+import { useHover } from "react-aria";
+import { Css, Palette } from "src/Css";
+import { ButtonMenu, MenuItem } from "./ButtonMenu";
+import { Tag, TagType } from "./Tag";
+
+export type TableCardType = "card" | "list";
+
+type CardTag = {
+  text: string;
+  type?: TagType;
+};
+
+export interface TableCardProps {
+  title: string;
+  subtitle: string;
+  metadata: ReactNode;
+  image: string;
+  // contain displays entire image, cover fills the space
+  imageFit?: "contain" | "cover";
+  type?: TableCardType;
+  bordered?: boolean;
+  disabled?: boolean;
+  buttonMenuItems?: MenuItem[];
+  tag?: CardTag;
+}
+
+export function TableCard(props: TableCardProps) {
```

Comment:
I was wondering about the "Table" naming for this component in the Figma so added a comment for Oz - seems like these cards aren't actually related to tables?

Reply:
good callout on the name. I'll wait to see if Stephen has input on what to call this

Reply:
I renamed the component to `Card` instead of `TableCard`